packages/flutter_tools/test/src/context.dart

```diff
@@ -55,7 +56,8 @@ void _defaultInitializeContext(AppContext testContext) {
     })
     ..putIfAbsent(SimControl, () => new MockSimControl())
     ..putIfAbsent(Usage, () => new MockUsage())
-    ..putIfAbsent(FlutterVersion, () => new MockFlutterVersion());
+    ..putIfAbsent(FlutterVersion, () => new MockFlutterVersion())
+    ..putIfAbsent(Clock, () => new Clock());
```

Comment:
Ah ya. I was gonna switch everything without breaking the existing tests but there were too many. Done. We can swap the others as needed.